.github/workflows/docs.yml

```diff
@@ -2,7 +2,7 @@ name: Docs
 
 on:
   push:
-    branches: [ master, release-0.6.3 ]
+    branches: [ master, release-0.6.3, vb/marimo ]
```

Comment:
This addition is only to build the `wasm/` folder using this branch, and should go away before we do an actual merge.